validator/test/fixtures/resource_value_pattern/constraints/data.yaml

```diff
@@ -0,0 +1,123 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+list:
+  - apiVersion: constraints.gatekeeper.sh/v1alpha1
+    kind: GCPResourceValuePatternConstraintV1
+    metadata:
+      name: required_billing_id_on_project
+    spec:
+      severity: high
+      match:
+        gcp:
+          target: ["organization/*"]
+          exclude: [] # optional, default is no exclusions
+      parameters:
+        mode: allowlist
+        asset_types:
+          - "cloudresourcemanager.googleapis.com/Project"
+        field_name: "labels.billing_id"
+        pattern: "^\\d{4}$"
+        required: true
```

Comment:
"optional: true" may have better UX since required is then implied in absence of the optional param.

Reply:
That makes sense- have changed to `optional`. Thanks!